### ISSUE TITLE
Fix Queries for the HomeworkView.php

### ISSUE THREAD

**Issue by prestoncarman** [wwu-cs/Submitty#10]
**Describe the bug**
Should not execute queries in the view.

```
app/views/submission/HomeworkView.php:  $students = $this->core->getQueries()->getAllUsers();
app/views/submission/HomeworkView.php:  foreach ($this->core->getQueries()->getGradedGradeables([$gradeable], $student_ids) as $gg) {
app/views/submission/HomeworkView.php:  $is_valid = null !== $this->core->getQueries()->getUserByIdOrNumericId($id);
app/views/submission/HomeworkView.php:  $grade_inquiry_posts = $this->core->getQueries()->getRegradeDiscussions($grade_inquiries);
app/views/submission/HomeworkView.php:  $is_staff = $this->core->getQueries()->isStaffPost($post['user_id']);
app/views/submission/HomeworkView.php:  $name = $this->core->getQueries()->getUserById($post['user_id'])->getDisplayedFirstName();
```

**Expected behavior**
The expected behavior is to execute the queries in the controller and pass the result to the view.

**Comment by PlasmaIntec** [comment]
Fixed by https://github.com/wwu-cs/Submitty/pull/12/commits/7a2b920a0ed19bd23ae6e2311176ed544760f2a7

**Comment by danielarlt** [comment]
> Fixed by [7a2b920](https://github.com/wwu-cs/Submitty/commit/7a2b920a0ed19bd23ae6e2311176ed544760f2a7)

This is not fixed by that commit.

**Comment by andrewdbinder** [comment]
Completed with PR #61
